index.html

```diff
@@ -1382,8 +1384,8 @@ <h4>Ed25519VerificationKey2020</h4>
             <p>
   The `publicKeyMultibase` property of the verification method MUST be
   a public key encoded according to [[?MULTICODEC]] and formatted according to
-  [[MULTIBASE]]. The multicodec encoding of an Ed25519 public key is the two-byte
-  prefix `0xed01` followed by the 32-byte public key data. The 34 byte
+  [[?MULTIBASE]]. The multicodec encoding of an Ed25519 public key is the
+  two-byte prefix `0xed01` followed by the 32-byte public key data. The 34 byte
```

Comment:
Presuming `0xED01` is equivalent to `0xed01` as the `two-byte prefix`, it's easier to read properly with the hex digits in uppercase.
```suggestion
  two-byte prefix `0xED01` followed by the 32-byte public key data. The 34-byte
```

Reply:
Not taking this change since we use lowercase for all hex values across all the DI specs presently. I'm not convinced that uppercase hex values are a significant enough improvement to do this change across every DI spec.

Reply:
For myself, I find `0x` blends with following lowercase and I have to read it at least twice to see it as hex. I won't lie down in the road on it, but I find uppercase *much* clearer in most cases.

Reply:
(I also note that while they don't _say_ anything about uppercase vs lowercase hex values anywhere else, all [Unicode code points in the W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/#Unicode) — which are the only hex in the manual — _are_ uppercase, `U+XXXX`.)